src/Filters/organisms/FilterBar.tsx

```diff
@@ -45,16 +53,16 @@ const createDropdownFilters = (
   dropdownsConfig: IFilterDropdownConfig[],
   filtersValues: IFilterResult[],
   allowMultiFilter: boolean,
-  handleSelected: (newValue: IFilterValue, filterId: string) => void
+  handleDropdownsSelected: (newValue: IFilterValue, filterId: string) => void
 ): IFilterDropdownExt[] => {
 
   const dropdownFilters: IFilterDropdownExt[] = [];
   dropdownsConfig.forEach(dropdown => {
     const filter = filtersValues.find(filter => filter.id === dropdown.id);
     if (filter) {
-      const selected = filter.selected;
-      const onSelect = (newSelection: IFilterValue) => { handleSelected(newSelection, filter.id); };
-      let disabled = getDisableValue(filtersValues, allowMultiFilter, filter);
+      const selected = Array.isArray(filter.selected) || isFilterItem(filter.selected) ? filter.selected : null;
```

Comment:
Having a '||' followed by a ternary in the same line is a little confusing. Is there a clearer way to define things here?

Reply:
I will update with identation